### DISCUSSION TITLE
Database Log With Custom Logging

### DISCUSSION THREAD

**Discussion by iqionly** [opcodesio/log-viewer#361] [Ideas]
I think this is new idea or not, but I have to see my custom Log data in database MySQL in my project.

Example my custom log, something like this:
![image](https://github.com/opcodesio/log-viewer/assets/36873896/5be7c60f-c65a-4811-94fd-c4d6fa996c0e)

This will showed in log viewer UI with other log files. If it can configure somehow, how to do that in the new version log viewer

Thank you

**Comment by arukompas** [comment]
hey @iqionly ,

at the moment Log Viewer can only read logs from raw text files on disk. No other drivers are supported at the moment (MySQL, S3, Redis, or any other).

The custom log parsers refer to the different log formats available, but they should still be on the disk as raw text files.

So what you're suggesting would be a feature request, then, but I can't promise a solution any time soon :) 

Hope this answers your question!
